src/axom/primal/operators/compute_bounding_box.hpp

```diff
@@ -121,6 +122,18 @@ AXOM_HOST_DEVICE BoundingBox<T, NDIMS> compute_bounding_box(
   return res;
 }
 
+/*!
+ * \brief Creates a bounding box around a Quadrilateral
+ * \accelerated
+ * \param [in] quad The Quadrilateral
+ */
+template <typename T, int NDIMS>
+AXOM_HOST_DEVICE BoundingBox<T, NDIMS> compute_bounding_box(
+  const Quadrilateral<T, NDIMS> &quad)
+{
+  return BoundingBox<T, NDIMS> {quad[0], quad[1], quad[2], quad[3]};
```

Comment:
This is nice!

Note: We should probably use initializer lists for the other primitives w/ a constant number of vertices, e.g. `Triangle`, `Hexahedron` -- it might be more efficient in some cases.
(you don't need to make this change either).

Reply:
It was simple enough, so I added it to this merge request.